src/hotkeys.cpp

```diff
@@ -26,27 +45,36 @@ static std::vector<HotkeyList*> *_hotkey_lists = nullptr;
 
 /** String representation of a keycode */
 struct KeycodeNames {
-	const char *name;       ///< Name of the keycode
-	WindowKeyCodes keycode; ///< The keycode
+	const char *name;         ///< Name of the keycode
+	WindowKeyCodes keycode;   ///< The keycode
+	const char *display_name; ///< The "pretty" name used to display the hotkey in the GUI
+
+	KeycodeNames(const char *name, WindowKeyCodes keycode, const char *display_name = nullptr)
+	{
+		this->name = name;
+		this->keycode = keycode;
+		this->display_name = display_name;
+	}
 };
 
 /** Array of non-standard keycodes that can be used in the hotkeys config file. */
 static const KeycodeNames _keycode_to_name[] = {
-	{"SHIFT", WKC_SHIFT},
-	{"CTRL", WKC_CTRL},
-	{"ALT", WKC_ALT},
+	{"SHIFT", WKC_SHIFT, "Shift"},
+	{"CTRL", WKC_CTRL, "Ctrl"},
+	{"ALT", WKC_ALT, "Alt"},
```

Comment:
These need to be translatable strings. For example, in Germany the Ctrl key is called *Strg*. Also keep in mind that many hotkeys on Mac systems use the *Cmd* key where other platforms use Ctrl.

Reply:
As pointed out by dP in the development discord channel, I need to do quite a bit more work on the translating / formatting parts. I will take your comments into account as well.